src/components/ADempiere/Form/PriceChecking/index.vue

```diff
@@ -58,22 +58,44 @@
               <div class="product-price-base">
                 {{ $t('form.priceChecking.basePrice') }}
                 <span class="amount">
-                  {{ formatPrice(productPrice.priceBase, productPrice.currency.iSOCode) }}
+                  {{
```

Comment:
Just can see change for soiurce code with the same functionality, please let it without changes